### ISSUE TITLE
Lockscreen option doesn't work

### ISSUE THREAD

**Issue by EthabTab** [patzly/doodle-android#95]
I'm using a mi 11 ultra. Whenever I try and set the wallpaper to both homescreen and lockscreen, it only sets the wallpaper on the homescreen and resets my lockscreen wallpaper to the default one. The wallpaper works perfectly fine on my homescreen.

**Comment by TaivasJumala** [comment]
TL;DR, just follow steps in numbers below.


Hey bro, I'm using Xiaomi Mi 11 (Venus). So theoretically, we use the same
Android System variant. Bellowing was what I met, and how I fixed it.

Actually, the very first time I set a wallpaper in this app, the app will ask
me that if I want to set the wallpaper for home screen and lock screen or only
Home screen. And it actually set lock screen for me. Later on, somehow the lock
screen returned to the default one.

Basing on that, I tried following steps you may take as reference:
(The option name mentioned below may not exact same as you are, because I'm not
using an English system, you should be able to figure out what it means.)

1. Long press Doodle's app icon, on pop up bubble choose app info.
2. On the bottom bar, select to clear its cache.
3. Set wallpaper in the app, be carefully set wallpaper for home and lock
   screen.
4. Again, navigate to the app info page, enable Doodle to automatically start.
5. Besides, you need to change the power save option for Doodle to no limit.

The logic behind those steps is that MIUI may support live wallpaper for home
screen but not supports for lock screen. It needs Doodle to maintain it
background.

If my steps not help, you may take a look at guides below:

Xiaomi | Don't kill my app!
https://dontkillmyapp.com/xiaomi

**Comment by patzly** [comment]
@TaivasJumala Thank you so much for your help! I will include the steps in the in-app help as well.
